src/App/Documentation/components/ActionList/index.js

```diff
@@ -0,0 +1,97 @@
+import React, { Component } from "react";
+
+import { ComponentPreview, DocToc, Icon } from "#";
+import actionList from "$/px-script/action-list";
+
+const Overview = () => (
+    <>
+        <h2 id="overview">Overview</h2>
+        <p>Action lists.</p>
+        <ComponentPreview language="html" showCasePanel={true} codeFigure={true}>
+            <div className="action-list">{"\n"}
+                <Icon icon="more_vert" />{"\n"}
+                <div className="action-menu">{"\n"}
+                    <a href="#" onClick={e => e.preventDefault()}><Icon icon="bookmark"/>Add bookmark</a>{"\n"}
+                    <a href="#" onClick={e => e.preventDefault()}><Icon icon="business_center"/>Add client</a>{"\n"}
+                    <a href="#" onClick={e => e.preventDefault()}><Icon icon="add_circle"/>Add document</a>{"\n"}
+                    <a href="#" onClick={e => e.preventDefault()}><Icon icon="person_add"/>Add user</a>{"\n"}
+                </div>
+            </div>
+        </ComponentPreview>
+    </>
+);
+
+const Anchorpoints = () => (
+    <>
+        <h2 id="anchorpoints">Anchorpoints</h2>
+        <p>Customize anchorpoint... Top right is default.</p>
+        <h3>Top left</h3>
+        <p>Use class <code className="token property">.anchor-top-left</code> to anchor action-list button to the top left corner of the menu.</p>
```

Comment:
The other components typically use "position", perhaps this one should aswell?

Reply:
Yeah, but in this case I felt that anchorpoint might be better, but I might also be wrong. Im open to `position-bottom-left` if that feels more natural.